requre/storage.py

```diff
@@ -86,6 +87,71 @@ class DataTypes(Enum):
     Dict = 2
 
 
+class StorageKeyStrategy(Enum):
+    Full = 1
+    Simple = 2
+    SuperSimple = 3
+    Custom = 4
+    Default = Full
+
+    @staticmethod
+    def __get_base_keys_custom(func: Callable) -> List[Any]:
+        raise NotImplementedError(
+            "You have to redefine this method to use custom store format"
+        )
+
+    @staticmethod
+    def __get_base_keys_full(func: Callable) -> List[Any]:
+        output: List[str] = list()
+        # callers module list, to be able to separate requests for various services in one file
+        caller_list: List[str] = list()
+        for currnetframe in inspect.stack():
+            if inspect.getmodule(currnetframe[0]):
+                module_name = inspect.getmodule(currnetframe[0]).__name__
+                # avoid to store requre.storage to module stack
+                # backward compatibility issue
+                if module_name.startswith("requre.storage"):
+                    continue
+            else:
+                continue
```

Comment:
I would solve the `continue` as soon as possible to no need to go so deep (=to the right with indentation)...

```python
            if not inspect.getmodule(currnetframe[0]):
                continue

            module_name = inspect.getmodule(currnetframe[0]).__name__
            # avoid to store requre.storage to module stack
            # backward compatibility issue
            if module_name.startswith("requre.storage"):
                continue
```